paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentMethodsUI.kt

```diff
@@ -203,3 +216,37 @@ internal fun PaymentMethodUI(
         }
     }
 }
+
+@Composable
+private fun PaymentMethodIconUi(
+    iconRes: Int,
+    iconUrl: String?,
+    imageLoader: StripeImageLoader,
+    tintOnSelected: Boolean,
+    color: Color,
+) {
+    val colorFilter = if (tintOnSelected) ColorFilter.tint(color) else null
```

Comment:
Nit: Should this be remembered?

Reply:
This was copy/paste from above. I'll take a look and see if we can remember it.

Reply:
I think we need to use `tintOnSelected` (and probably `color`?) as the key here. Otherwise, the card icon stays tinted even when not selected.